### ISSUE TITLE
github-runner: configure.sh: Can not find 'ldd'.

### ISSUE THREAD

**Issue by RaelCappra** [NixOS/nixpkgs#205400]
### Describe the bug
Upon starting the `github-runner` service with no extra packages set, the runner configuration fails with `Can not find 'ldd'. Please install 'ldd' and try again.`

### Steps To Reproduce
1. Enable the github-runner option in your system configuration;
2. Set up the self-hosted runner for the corresponding repo;
3. Rebuild the system. 

### Expected behavior
The runner would configure and work as normal if set up correctly.

### Additional context
Adding `glibc` to the list of `extraPackages` results in the expected behavior.

### Notify maintainers
@veehaitch @newam @kfollesdal @aanderse

### Metadata

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.75-xanmod1, NixOS, 22.11 (Raccoon), 22.11.740.7a6a010c3a1`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.11.0`
 - channels(root): `"home-manager-22.11.tar.gz, nixos-22.11, nixos-unstable"`
 - channels(rr): `"home-manager-22.11.tar.gz"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

```

**Comment by newAM** [comment]
Will be fixed by #205137

**Comment by veehaitch** [comment]
… actually only by this fix of the fix: https://github.com/NixOS/nixpkgs/pull/205294

**Comment by newAM** [comment]
#205645 was merged, this is fixed on 22.11 and unstable, closing.
